File: dream-team-mkt-business/src/main/java/co/simplon/dreamteam/mkt/business/entities/ContactForm.java
package co.simplon.dreamteam.mkt.business.entities;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record ContactForm(@NotBlank @Size(max = 200) String corporateName, @NotBlank @Size(max = 200) String firstName,
		@NotBlank @Size(max = 200) String lastName, String email, String phoneNumber, @NotBlank @Size(max = 2000) String message) {

}

File: dream-team-mkt-presentation/src/services/contactForm.js

import { axiosClient } from "./axiosClient"

class ContactForm {

  static sendContactFormData(inputs) {
    const data = {
      corporateName: inputs.corporateName.trim(),
      firstName: inputs.firstName.trim(),
      lastName: inputs.lastName.trim(),
      email: inputs.email.trim(),
      phoneNumber: inputs.phoneNumber.trim(),
      message: inputs.message.trim()
    }
    axiosClient.post('/mail/send',
      {data }
      )
  }
}

export default ContactForm;
